pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -58,7 +58,7 @@ const (
 	// The minimum memory limit allocated to docker container: 150Mi
 	MinDockerMemoryLimit = 150 * 1024 * 1024
 
-	dockerProcessName     = "docker"
+	dockerProcessName     = "dockerd?"
```

Comment:
It seems same between a regex `"dockerd?"` and a list of `["dockerd", "docker"]`. The name regex is compiled at [procfs.Pidof](https://github.com/kubernetes/kubernetes/blob/master/pkg/util/procfs/procfs_linux.go#L99)